docs/site-example/i18n/en.po

```diff
@@ -17,9 +17,10 @@ msgstr ""
 
 msgid "gluon-config-mode:pubkey"
 msgstr ""
-"This is your Freifunk node's public key. The node won't be able to "
-"connect to the mesh VPN until the key has been registered on the Freifunk "
-"Duckburg servers. "
+"This is the public key of your Freifunk node. "
+"Your Node may ether connect via other nodes within range or "
```

Comment:
ether -> either, Node -> node

Wie wäre es mit: Your node may either connect to nearby nodes through wifi or to gateway servers via Mesh-VPN...